mqttclients/dotnet/MQTTnet.Client.Extensions/CommandServer.cs

```diff
@@ -1,26 +1,25 @@
-﻿using MQTTnet;
-using MQTTnet.Client;
-using MQTTnet.Server;
+﻿namespace MQTTnet.Client.Extensions;
 
-namespace MQTTnet.Client.Extensions;
-
-public abstract class CommandProducer<T, TResp>
+public abstract class CommandServer<T, TResp>
 {
     private readonly IMqttClient _mqttClient;
     private readonly string _commandName;
 
     public Func<T, Task<TResp>>? OnCommandReceived { get; set; }
 
-    private string _requestTopic = string.Empty;
+    private readonly string _requestTopic = string.Empty;
 
     private readonly IMessageSerializer _serializer;
 
-    public CommandProducer(IMqttClient mqttClient, string cmdName, IMessageSerializer serializer)
+    public CommandServer(IMqttClient mqttClient, string cmdName, IMessageSerializer serializer)
     {
         _mqttClient = mqttClient;
         _commandName = cmdName;
         _serializer = serializer;
 
+        var rta = GetType().GetCustomAttributes(true).OfType<RequestTopicAttribute>().FirstOrDefault();
```

Comment:
In general, the managed team's standard for `var` is that it's only to be used when the type is very obvious, for example:

`var deviceClient = new DeviceClient()`

In cases where a new type is being returned from a method, or any other scenario where the type is not very obvious, we declare the type explicitly to avoid errors related to invalid types if that method's return type were to change.